index.html

```diff
@@ -105,6 +106,27 @@ <h3>
           reaches its maximum capacity and gets flushed.
         </p>
       </dd>
+      <dt>
+        <dfn><code>relativePacketArrivalDelay</code></dfn> of type
+        <span class="idlMemberType"><a href="https://heycam.github.io/webidl/#idl-double">double</a></span>
+      </dt>
+      <dd>
+        <p>
+          Experimental stat which will be available under origin trial in
+          Chrome. It is the sum, in seconds, of the difference between the
+          expected packet arrival time, based on the packets we have seen so
+          far, and the actual arrival time when a packet is inserted in the
+          jitter buffer. Expected arrival time is the difference in send time
+          between the received packet and a reference packet previously
```

Comment:
And the referenced packet? What does that mean? The time since it was received? Or the time it took for it to travel through the jitter buffer?